### PR TITLE
Add link to service response docs in conversation.markdown

### PR DESCRIPTION
**Pull request by jjbankert**
## Proposed change
Add link to service response docs in conversation.markdown

## Type of change
- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
- Link to parent pull request in the codebase: **unsure**
- Link to parent pull request in the Brands repository: **unsure**
- This PR fixes or closes issue: https://community.home-assistant.io/t/2023-7-responding-services/588733/10

## Checklist
- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/conversation.markdown [hunk 1]
```diff
@@ -167,7 +167,7 @@ Send a message to a conversation agent for processing.
 | `language`                 | yes      | Language of the text |
 | `agent_id`                 | yes      | ID of conversation agent to use |
 
-This service is able to return a response value. The response is the same response as for the
+This service is able to return [Response Data](https://www.home-assistant.io/docs/scripts/service-calls/#use-templates-to-handle-response-data). The response is the same response as for the
```
Comment (c0ffeeca7) [inline source/_integrations/conversation.markdown]:
```suggestion
This service is able to return [response data](https://www.home-assistant.io/docs/scripts/service-calls/#use-templates-to-handle-response-data). The response is the same response as for the
```

**Review by c0ffeeca7 — APPROVED** [review]
Nice! Thank you, @jjbankert :+1:

File: source/_integrations/conversation.markdown [hunk 2]
```diff
@@ -167,7 +167,7 @@ Send a message to a conversation agent for processing.
 | `language`                 | yes      | Language of the text |
 | `agent_id`                 | yes      | ID of conversation agent to use |
 
-This service is able to return a response value. The response is the same response as for the
+This service is able to return [response data](https://www.home-assistant.io/docs/scripts/service-calls/#use-templates-to-handle-response-data). The response is the same response as for the
```
Comment (frenck) [inline source/_integrations/conversation.markdown:170]: This link is not correct, is is a full link and should have been a link relative from the root.

If this was tested... you would have noticed you'd be navigating away from your development environment.

Please create an PR to correct this.

../Frenck

/CC @c0ffeeca7

Comment (c0ffeeca7) [reply source/_integrations/conversation.markdown:170]: Thank you! Will create a PR to fix it.

**Pull request merged**